eng/pipelines/templates/steps/analyze.yml

```diff
@@ -26,6 +26,12 @@ steps:
      scriptPath: 'scripts/devops_tasks/verify_change_log.py'
      arguments: '"${{ parameters.BuildTargetingString }}" --service=${{parameters.ServiceDirectory}}'
 
+  - script: |
+      cd eng/versioning
+      pip install -r requirements.txt
+      python find_invalid_versions.py --always-succeed --service-directory ${{parameters.ServiceDirectory}}
```

Comment:
Can you also include a comment here as to why we are set to always succeed and link to the issue to undo that?